pkg/controllers/machine/registration.go

```diff
@@ -44,7 +44,7 @@ func (r *Registration) Reconcile(ctx context.Context, machine *v1alpha5.Machine)
 	if err != nil {
 		if IsNodeNotFoundError(err) {
 			machine.StatusConditions().MarkFalse(v1alpha5.MachineRegistered, "NodeNotFound", "Node not registered with cluster")
-			return reconcile.Result{RequeueAfter: registrationTTL}, nil // Requeue later to check up to the registration timeout
+			return reconcile.Result{}, nil // Requeue later to check up to the registration timeout
```

Comment:
Is this comment accurate? Doesn't look like we'd be requeueing here.

Reply:
Good catch